mu-plugins/blocks/language-suggest/src/front.js

```diff
@@ -1,14 +1,17 @@
+/* global LanguageSuggestData */
+
 const init = () => {
 	const container = document.querySelector( '.wp-block-wporg-language-suggest' );
 
 	if ( ! container ) {
 		return;
 	}
 
-	const endpoint =
-		container.dataset.endpoint ||
-		'https://wordpress.org/lang-guess/lang-guess-ajax.php?uri=' +
-			encodeURIComponent( window.location.pathname );
+	const endpoint = new URL(
+		container.dataset.endpoint || 'https://wordpress.org/lang-guess/lang-guess-ajax.php'
+	);
+	endpoint.searchParams.set( 'uri', encodeURIComponent( window.location.pathname ) );
+	endpoint.searchParams.set( 'locale', encodeURIComponent( LanguageSuggestData.locale ) );
```

Comment:
I think `set()` already encodes, but as none of the characters need encoding, I think this is safe to include.

Reply:
yeah, you're right, none of the chars need it and `set()` already does enough encoding.
removed: 17fcac1